### PR TITLE
Simplified Chinese Translation For 1.18.2

### PR DESCRIPTION
**Pull request by Cactusstudent**

### CODE REVIEW

**Comment by KnightMiner** [comment]
Make sure you test out the modifier pages with the icons to make sure the text still fits, I had to shorten. the text on the English pages. Chinese does generally use fewer characters to express the same idea so it's probably fine but the possibility is why I did not just add icons to other languages.

**Comment by Cactusstudent** [comment]
Sorry for being late, have tested in the build version (.148), seem everything works properly. Also in the latest build version

**Comment by KnightMiner** [comment]
Added one last thing to the book, forgot pages on tinted glass. I'll probably merge this before a build whether you add those or not, but I think its the only thing not translated since you caught the other last minute additions.

**Comment by Cactusstudent** [comment]
~Ok, thanks, I'll update this now~
Have updated now, works fine in the latest build (.151)

File: src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/tools/staff/tconstruct_ender_staff.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   "text": [
-    { "text": "末影黏液手杖是一种高触及距离的黏液手杖。" }
+    { "text": "末影黏液手杖是一种高触及距离和力量的黏液手杖。" }
   ],
   "properties": [
     "1520 耐久度",
-    "40% 使用时移速",
+    "3 弹射物力量",
+    "0.5 精确度",
+    "120 格挡角度",
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/tools/staff/tconstruct_ender_staff.json]:
```suggestion
    "120° 格挡角度",
```

File: src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/language.lang [hunk 2]
```diff
@@ -7,8 +7,8 @@ ranged_materials.subtext=这些材料可以放在部件制造台中，用于制
 
 tools=工具
 
-armor=旅行者套装
-armor.subtext=你可以在工作站中合成旅行者套装！这套装备虽然不能提供最好的防御，但它却拥有很多升 级功能。你会创造出什么有趣的东西来呢？
+armor=基础盔甲
+armor.subtext=基础盔甲无需通过猎杀获取资源或是建造任何机械。这套装备虽然只能提供有限的防御，但它却拥有很多升级功能。你会创造出什么有趣的东西来呢？
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/materials_and_you/zh_cn/language.lang]:
```suggestion
armor.subtext=基础盔甲无需通过猎杀获取资源或是建造任何机械即可制作。这套盔甲虽然只能提供有限的防御，但它却拥有很多升级功能。你会创造出什么有趣的东西来呢？
```

File: src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/leggings/tconstruct_wetting.json [hunk 3]
```diff
@@ -3,14 +3,12 @@
...
-    {
-      "text": "手持工具右击储罐以填充。", "paragraph": true
     }
   ],
   "effects": [
     "在受伤时应用流体效果",
     "单个等级",
-    "需要1个能力槽"
+    "需要1个能力槽",
+    "手持工具右击储罐以填充。"
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/leggings/tconstruct_wetting.json]:
```suggestion
    "手持工具右击储罐以填充"
```

File: src/main/resources/assets/tconstruct/book/puny_smelting/zh_cn/language.lang [hunk 4]
```diff
@@ -9,6 +9,9 @@ ranged_materials.subtext=这些材料可以铸造成像弓那样的远程工具
 
 further_reading=延伸阅读
 
+armor=入门盔甲
+armor.subtext=借由铜这种早期可以获得的资源，你可以在工作站中制造出例如旅行者套装这样的入门盔甲！这套盔甲，这套盔甲虽然不能提供最好的防御，但你却有很多升级这套盔甲的机会！
```
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/puny_smelting/zh_cn/language.lang]:
```suggestion
armor.subtext=借由铜这种早期可以获得的资源，你可以在工作站中制造出例如旅行者套装这样的入门盔甲！这套盔甲虽然不能提供最好的防御，但你却有很多升级这套盔甲的机会！
```

**Pull request merged**